lang/node-mozilla-iot-gateway/Makefile

```diff
@@ -44,6 +52,33 @@ define Package/node-mozilla-iot-gateway/description
   and defining a standard data model and APIs to make them interoperable.
 endef
 
+define Download/ursa
+  FILE:=$(URSA_SOURCE)
+  URL:=https://registry.yarnpkg.com/ursa/-/$(FILE)
+  HASH:=65516fc99958011572ef26ef02a3c84a004e47aabfd5b8aa46abb10481ae7153
+endef
+$(eval $(call Download,ursa))
```

Comment:
This really looks like it should be a separate package, so it gets all the regular package downloading treatment and all the dependency tracking like normal, instead of the hackjob you've got below.

Reply:
I agree with you.  However, there are 1741 packages in that situation--if you look at the compile log, there are 2 lines, adding 583 and 1203 packages.  It's a 42MB ipk file.  It installs a copy of node-serialport package as well, despite the fact it is a separate package.  The package was added to the feed as is.  I just wanted was to make it compile with openssl 1.1, and didn't fix anything else.